docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -211,10 +212,11 @@ Before you can configure the server to use the JDBC persistence layer implementa
 
 The IBM FHIR Server is delivered with a default configuration that is already configured to use the JDBC persistence layer implementation with an Embedded Derby database. This provides the easiest out-of-the-box experience since it requires very little setup.
 
-The IBM FHIR Server persistence configuration is split between `fhir-server-config.json` and the Liberty `server.xml + configDropins`.
+The IBM FHIR Server persistence configuration is split between `fhir-server-config.json` and the Liberty `server.xml` and `configDropins`.
 
 Within `fhir-server-config.json`, the value of the `fhirServer/persistence/factoryClassname` is used to instantiate a FHIRPersistence object. By default, the server is configured to use the FHIRPersistenceJDBCFactory:
-```
+
+``` json
```

Comment:
the `json` hint seems to cause the github editor some trouble, since technically the snippet isn't valid json

Reply:
it's the elipsis + json